src/controllers/controller.doughnut.js

```diff
@@ -76,7 +76,7 @@ export default class DoughnutController extends DatasetController {
 
   static descriptors = {
     _scriptable: (name) => name !== 'spacing',
-    _indexable: (name) => name !== 'spacing',
+    _indexable: (name) => name !== 'spacing' && !name.startsWith('borderDash') && !name.startsWith('hoverBorderDash'),
```

Comment:
why `startsWith`? and probably does not need `hoverBorderDash`, as its returned as `borderDash` when hovered.

Reply:
@kurkle I used `startsWith` to manage also `borderDashOffset` at the say way of `borderDah` because the 2 options are strictly related.

Reply:
> probably does not need `hoverBorderDash`, as its returned as `borderDash` when hovered.

It seems you're right. I was convinced that the indexable policy is ignored for hover if not specified but it doesn't seem so. I'll submit another PR to remove it.